projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts

```diff
@@ -250,4 +272,108 @@ describe('AddToCartComponent', () => {
     const quantityEl = el.query(By.css('.quantity'));
     expect(quantityEl).toBeNull();
   });
+
+
+
+  describe('Inventory Display test', () => {
+
+    it('should display inventory quantity when enabled', () => {
+
+      const currentMockProduct: Product = {
```

Comment:
let's put currentMockProduct in the describe above and reuse it for every it case